template/web/src/modules/Scrolly/ScrollySection/index.tsx

```diff
@@ -1,11 +1,14 @@
 import React, { FC, useEffect } from 'react'
 import { graphql } from 'gatsby'
 import Box from '@/atoms/Box'
-import ModulesLoop from '@/modules'
-import ImageModule from '@/modules/Image'
+import ScrollyModulesLoop from '@/modules/Scrolly'
 import { SanityModuleScrollySection } from 'web/types/graphql-types'
+import { gsap } from 'gsap'
+import { ScrollTrigger } from 'gsap/dist/ScrollTrigger'
 
-const ScrollySectionModule: FC = (props: SanityModuleScrollySection) => {
+gsap.registerPlugin(ScrollTrigger)
```

Comment:
is this the right place to run this?

1. multiple sections means it will be registered multiple times
2. no effect binding means it will run with every re-render